app/views/admin/loans/timeline/table/_timeline.html.slim

```diff
@@ -1,9 +1,12 @@
+- max_depth = @loan.root_timeline_entry.max_descendant_depth
 .timeline-table
   - if @loan.root_timeline_entry
     - if @loan.root_timeline_entry.is_a?(ProjectGroup)
       table.table
-        = render "admin/loans/timeline/table/group", entry: @loan.root_timeline_entry, is_first: true,
-          max_depth: @loan.root_timeline_entry.max_descendant_depth
+        = render "admin/loans/timeline/table/header", max_depth: max_depth
+        tbody
```

Comment:
I can also move tbody into the code for the very first group, if that is cleaner.